src/ape/api/compiler.py

```diff
@@ -14,7 +14,12 @@ class CompilerAPI:
     @property
     @abstractmethod
     def name(self) -> str:
-        ...
+        """
+        The name of the compiler.
```

Comment:
Might be good to add how this is used in the codebase, which.... I don't know if it is lol

Reply:
`name` sounds like something that can be in our base API class (have the pydantic work in mind a little)

Reply:
Yes but I don't think it does anything right now

I think the target might have been for the `compilers` field of the manfiest

Reply:
- [ ] let's remove this doc str

Reply:
So should i just remove the comment?